apps/prairielearn/src/question-servers/calculation-subprocess.js

```diff
@@ -36,20 +38,29 @@ async function prepareChunksIfNeeded(question, course) {
 
 async function callFunction(func, question_course, question, inputData) {
   await prepareChunksIfNeeded(question, question_course);
-  const coursePath = chunks.getRuntimeDirectoryForCourse(question_course);
+
+  const courseHostPath = chunks.getRuntimeDirectoryForCourse(question_course);
+  const courseRuntimePath = config.workersExecutionMode === 'native' ? courseHostPath : '/course';
```

Comment:
This is the usual problem we've faced elsewhere, where we need to carefully handle paths when running natively vs. in a container.

This is the reason that grading hit the error case in production. That said, `generate()` was *also* erroring, but that never leaked back to users because of the parallel-run pattern, so yay for that!